homeassistant/components/bond/light.py

```diff
@@ -83,6 +104,29 @@ class BondBaseLight(BondEntity, LightEntity):
 
     _attr_supported_features = 0
 
+    async def async_set_state_belief(self, **kwargs: Any) -> None:
+        """Set the belief state of the light."""
+        brightness = kwargs.get(ATTR_BRIGHTNESS)
+        if brightness is not None:
+            if not self._device.supports_set_brightness():
+                raise TemplateError(
+                    Exception("This device does not support brightness")
+                )
+            if brightness == 0:
+                await self._hub.bond.action(
+                    self._device.device_id, Action.set_light_state_belief(False)
+                )
+            else:
+                await self._hub.bond.action(
+                    self._device.device_id,
+                    Action.set_brightness_belief(round((brightness * 100) / 255)),
+                )
+        else:
+            state = kwargs.get(ATTR_POWER_STATE)
```

Comment:
If they don't provide a value, this will be set to false. That might not be what the user expects

Reply:
I will add a raise there.
            if state is None:
                raise TemplateError(
                    Exception(
                        f"Either {ATTR_BRIGHTNESS} or {ATTR_POWER_STATE} is required.  Neither provided."
                    )
                )